Sources/SwifterSwift/UIKit/UIViewControllerExtensions.swift

```diff
@@ -111,6 +111,24 @@ public extension UIViewController {
         removeFromParent()
         view.removeFromSuperview()
     }
+    
+    /// SwifterSwift: Helper method to set tabBarItem when UIViewController is added to a UITabBarViewController with image from SF Symbol
+    ///  - Example: viewController.setTabBarImage(systemName: "message.fill", title: "Messages")
+    ///
+    ///  - Parameters:
+    ///     - imageName: the SF Symbol name.
+    ///     - configuration: UIImage.SymbolConfiguration by defualt is scaled to large
+    ///     - title: the tab bar title.
+    @available(iOS 13.0, *)  @available(tvOS 13.0, *)
+    func setTabBarImage(
+        systemName: String,
+        configuration: UIImage.SymbolConfiguration = .init(scale: .large),
+        title: String? = nil) {
+        guard let image = UIImage(systemName: systemName, withConfiguration: configuration) else {
+            return
+        }
+            tabBarItem = UITabBarItem(title: title ?? tabBarItem.title, image: image, tag: tabBarItem.tag)
```

Comment:
Why is this not aligned with the `guard`? Can you fix the formatting? Xcode can do it for you by highlighting and pressing ctrl+`i`